src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs

```diff
@@ -0,0 +1,8 @@
+namespace Windows.ApplicationModel.Activation
+{
+	public partial class ToastNotificationActivatedEventArgs : global::Windows.ApplicationModel.Activation.IToastNotificationActivatedEventArgs, global::Windows.ApplicationModel.Activation.IActivatedEventArgs, global::Windows.ApplicationModel.Activation.IActivatedEventArgsWithUser, global::Windows.ApplicationModel.Activation.IApplicationViewActivatedEventArgs

```

Comment:
Remove the namespaces and add a using instead 👍

Reply:
Done